driver/chardev.c

```diff
@@ -40,6 +39,7 @@
 #include <linux/uaccess.h>
 #include <stddef.h>
 #include <uapi/linux/kfd_ioctl.h>
+#include "chardev.h"
```

Comment:
You might need to add a blank line, otherwise clang-format wants alphabetical order.

Reply:
Done